### PR TITLE
 [Duplicate of #6155] Getting started with Nessus Scanner

### LINKED ISSUE #5934: [Security] Getting started with Nessus scanner
#### Proposed title of article
Getting started with Nessus Scanner

#### Proposed article introduction
In this tutorial, I will be introducing you to the Nessus scanner and how to be a good Pen-tester using Nessus.
Nessus is a Web vulnerability scanner that allows you to scan your server/Website for vulnerabilities and provide tips on how to fix them. This is open-source software that is available for all, it is available on Windows but it comes inbuilt in Kali-Linux which makes things even easier.

#### Key takeaways 
The reader should be able to
- Install Nessus and know how to use it.

#### Article quality
Most sites covering the same topic only show you how the installation works and how to scan for vulnerabilities, but how to fix this vulnerabilities with nessus they don't show you.
So I will hold you by the hand and show you different ways to help secure a real site with Nessus after a successful scanning is done. You will be using Nessus to find, fix and update servers to increase your security levels.

### PR DESCRIPTION
**Pull request by Adespscientist**
This closes #5934

### CODE REVIEW

**Comment by Adespscientist** [comment]
@Bayler I purchased the premium version of Grammarly to correct all grammatical errors and fix all punctuations too. Check it out.

File: content/articles/getting-started-with-nessus-scanner/index.md [hunk 1]
```diff
@@ -0,0 +1,218 @@
...
+
+- Paste the activation code into the activation box and click continue.
+
+- Provide your username and password for login
+
+- Nessus will start the initializing process, which will take a couple of minutes; let it install while you get a drink or coffee.
+
+- That took forever! All right, we have loaded Nessus, and it is installed.
+
+![Metersploitable](metersploitable.png)
```
Comment (Bayler) [inline content/articles/getting-started-with-nessus-scanner/index.md:128]: This formatting for referencing an image is wrong. Please refer to this [article](https://github.com/section-engineering-education/engineering-education/pull/6394/files) on line 124 to see the correct way.

**Comment by Adespscientist** [comment]
@Bayler, I have made necessary changes sir.

**Comment by Adespscientist** [comment]
I have made changes to the image referencing

**Comment by Adespscientist** [comment]
@Bayler,  I have learnt alot from all the commit. I will sure work and improve on my technical writings.

**Pull request closed without merging**